diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -3151,6 +3168,10 @@ export class ContentGame extends React.Component {
                 suggestionTypeDisplay.push("commentary");
         }
 
+        if (!fullSuggestionComponent && !partialSuggestionComponent && !suggestionTypeDisplay.includes("move")) {
+            return null;
+        }
+
```

Comment:
This statement prevents displaying information about advice provided. Is that intentional? I feel that message should always be displayed, including cases where no move advice is provided.

Reply:
The idea is that if there is no move advice, then it should not take any space in the interface and player would play as if there is no advice.

Reply:
But they could still be getting message advice, commentary advice, etc.

Reply:
yes. This part only concerns move advice.

Reply:
I mean that the message "You are getting advice this turn: ..." won't be displayed if only move advice is provided.

Reply:
For that, my plan is to move them under the orders tab in a separate PR, as information about advice should not depends on an advice component

Reply:
See [#64 ](https://github.com/ALLAN-DIP/diplomacy/pull/64)